resources.go

```diff
@@ -964,27 +964,32 @@ func Provider() tfbridge.ProviderInfo {
 								// it's possible (likely) during previews that the location will be unknown, so
 								// we special logic to propagate likewise unknown location values.
 								if rg, has := res.Properties["resourceGroupName"]; has {
-									rgName := rg.StringValue()
-									rgRegion, has := rgRegionMap[rgName]
-									if !has {
-										rgRes := p.ResourcesMap["azurerm_resource_group"]
-										contract.Assert(rgRes != nil)
-										rgData := rgRes.Data(&terraform.InstanceState{
-											// Mock up a URI with the relevant pieces, so that we can read back
-											// the resource group's location information.
-											ID: fmt.Sprintf("/subscriptions/_/resourceGroups/%s", rg.StringValue()),
-										})
-										if err := rgRes.Read(rgData, p.Meta()); err != nil {
-											return nil, err
-										}
-										if rgData.Id() == "" {
-											rgRegion = config.UnknownVariableValue
-										} else {
-											rgRegion = azure.NormalizeLocation(rgData.Get("location"))
+									if rg.IsComputed() || rg.IsOutput() {
+										return "[computed]", nil
```

Comment:
I *think* that it would be better to not magic up a value here - and to instead just ensure that we return `config.UnknownVariableValue`.  That should get replaced with something like `[computed]` at the CLI rendering layer - but will ensure that the value flows correctly through other layers.

Reply:
Great suggestion! This prints `output<string>` in the preview details.